lib/web/mfa.go

```diff
@@ -132,9 +133,10 @@ func (h *Handler) addMFADeviceHandle(w http.ResponseWriter, r *http.Request, par
 }
 
 type createAuthenticateChallengeRequest struct {
-	IsMFARequiredRequest *isMFARequiredRequest `json:"is_mfa_required_req"`
-	ChallengeScope       int                   `json:"challenge_scope"`
-	ChallengeAllowReuse  bool                  `json:"challenge_allow_reuse"`
+	IsMFARequiredRequest        *isMFARequiredRequest                 `json:"is_mfa_required_req"`
+	ChallengeScope              int                                   `json:"challenge_scope"`
+	ChallengeAllowReuse         bool                                  `json:"challenge_allow_reuse"`
+	UserVerificationRequirement *protocol.UserVerificationRequirement `json:"user_verification_requirement"`
```

Comment:
```suggestion
	UserVerificationRequirement string `json:"user_verification_requirement"`
```

Go won't magically coerce the unmarshaled values to the enum set, so this is just giving us a false sense of (type) safety. Plus, this is just a string on ChallengeExtensions.

Reply:
I can change it, though I'd like to point out that we already have some JSON-serializable structures that have fields of this type explicitly typed.

Reply:
Yes, please change this.

What other structs do we have?

Reply:
OK, will do.

Other structs:
https://github.com/gravitational/teleport/blob/master/lib/auth/webauthntypes/webauthn.go#L46
https://github.com/gravitational/teleport/blob/master/lib/auth/webauthntypes/webauthn.go#L193

Reply:
Structs within webauthntypes are fine.